### ISSUE TITLE
Docker image doesn't contain playback files for BigBlueButton 2.3 recordings

### ISSUE THREAD

**Issue by kepstin** [blindsidenetworks/scalelite#416]
The nginx frontend docker image is built using the BigBlueButton 2.2 packages to get the recording playback files, and as a result it's missing the playback files for BigBlueButton 2.3.

**Comment by atikulbd** [comment]
Install **BBB 2.2** by adding ***-v xenial-22*** option on a Ubuntu 16.04 server.
```
https://github.com/bigbluebutton/bbb-install
```
So, both Scalelite and BBB will comply.

**Comment by jfederico** [comment]
Since BBB 2.3 has not officially released as stable, we still don't have the player and other pieces for recordings that support 2.3 in the stable release.

There are however a set of extended images that can be used for this. See https://github.com/blindsidenetworks/scalelite/blob/master/docker-README.md

The change was planned for v1.1 and further, but there are extended images for 1.0 as 1.0.12.1. This is the way it will be done until 2.3 is released as stable.

Anyone can try it using one of the new extended images by editing your `/etc/default/scalelite` file, like this:

```
SCALELITE_REPO=blindsidenetwks
SCALELITE_TAG=v1.0.12.1-bionic230-amazonlinux
```

```
SCALELITE_REPO=blindsidenetwks
SCALELITE_TAG=v1.0.12.1-bionic230-alpine
```

**Comment by harika1258** [comment]
@jfederico 
This problem have been solved in 2.3.12 released now for Scalelite?
